data-esnext.js

```diff
@@ -1712,6 +1712,7 @@ exports.tests = [
     babel7corejs2: true,
     ie11: false,
     firefox52: false,
+    safaritp: {val: 'flagged', note_id: "safari-nullish", note_html: "You can flip it on like this: <code>__XPC_JSC_useNullishCoalescing=true && open -a 'Safari Technology Preview'</code>."},
```

Comment:
no &&, or it won’t work
```suggestion
    safaritp: {val: 'flagged', note_id: "safari-nullish", note_html: "You can flip it on like this: <code>__XPC_JSC_useNullishCoalescing=true open -a 'Safari Technology Preview'</code>."},
```

Reply:
Done